HousingSearchApi/V1/Infrastructure/ElasticSearchExtensions.cs

```diff
@@ -26,5 +27,29 @@ public static void ConfigureElasticSearch(this IServiceCollection services, ICon
 
             services.TryAddSingleton<IElasticClient>(esClient);
         }
+
+        /// <summary>
+        /// Applies Descending() or Ascending() method based on <paramref name="isDesc"/>
```

Comment:
There is another place to do this - check : https://github.com/LBHackney-IT/housing-search-api/blob/master/HousingSearchApi/V1/Interfaces/Sorting/SortFactory.cs . For person, should this still be as before, surname/firstname is already implemented. Alternatively add to what's already there. Thanks !

Reply:
This method is unused now. We can delete it

Reply:
Deleted